src/scry/hover_provider.cr

```diff
@@ -63,13 +63,47 @@ module Scry
     end
 
     private def response_with(contexts, range)
-      contents = ["| Expression | Type |", "| --- | --- |"]
-      contexts.each do |context|
+      content = vertical_align(contexts)
+      # content = horizontal_align(contexts)
+      hover_response(Hover.new(MarkupContent.new("markdown", content.join("\n")), range))
+    end
+
+    private def vertical_align(contexts)
+      contents = [] of String
+      contexts.each_with_index do |context, i|
+        contents << "**Context #{i + 1}**\n"
```

Comment:
If there is only one context we should hide the `Context 1/2/...`

Reply:
Done!